### PR TITLE
Fix layer naming

### PR DESCRIPTION
**Pull request by tomkoren21**

### CODE REVIEW

File: onnx2kerastl/converter.py [hunk 1]
```diff
@@ -209,7 +209,10 @@ def onnx_to_keras(onnx_model, input_names, name_policy=None, verbose=True, chang
             else:
                 keras_names = keras_names[0]
                 node_names.append(keras_names)
-
+            cleaned_node_name = node.name.replace("#", "_").rstrip("/").lstrip("/").replace(":", "_")
```
Comment (RanHomri) [inline onnx2kerastl/converter.py:212]: Maybe we can create some list with characters to remove and use regular expressions for that.
will be easier to maintain

**Review by RanHomri — APPROVED** [review]
Left one suggestion and found some typos.
other than that lokks great

File: onnx2kerastl/activation_layers.py [hunk 2]
```diff
@@ -22,7 +23,7 @@ def convert_relu(node, params, layers, lambda_func, node_name, keras_name):
 
     input_0 = ensure_tf_type(layers[node.input[0]], name="%s_const" % keras_name)
 
-    relu = keras.layers.Activation('relu', name=keras_name)
+    relu = keras.layers.Activation('relu', name=f"{params['cleaned_name']}_reli")
```
[the remark this answers is not in the document]
Comment (tomkoren21) [reply onnx2kerastl/activation_layers.py:26]: fixed

File: onnx2kerastl/convolution_layers.py [hunk 3]
```diff
@@ -118,7 +121,7 @@ def convert_conv(node, params, layers, lambda_func, node_name, keras_name):
 
         if padding:
             logger.debug('Paddings exist, add ZeroPadding layer')
-            padding_name = keras_name + '_pad'
+            padding_name = f"{params['cleaned_name']}" + 'conv_pad_1'
```
[the remark this answers is not in the document]
Comment (tomkoren21) [reply onnx2kerastl/convolution_layers.py:124]: fixed

**Pull request merged**